Emilgardis/twitch_types
# improve ci

- rustfmt config
- move org
- add .gitignore
- aliri 0.2.0
- add campaign id
- fmt
- bump lockfile
- add Unicode-DFS-2016 to allowed licenses
- release twitch_types 0.3.0
- Fix clippy lints
- easier releases
- release twitch_types 0.3.1
- move release job up and make it need the other steps
- test whole workspace
- add new creator goals
- release twitch_types 0.3.2
- add colors
- release twitch_types 0.3.3
- add some convenience functions to HexColor - NamedUserColor
- add clone to NamedUserColor
- release twitch_types 0.3.4
- add ClipId
- release twitch_types 0.3.5
- add some more From impls
- Add `IntoCow` trait
- fix release.toml
- release twitch_types 0.3.6
- add igdb_id
- zerofrom and arbitrary impls
- add getters for the parts of a time
- add numeric and ascii arb
- release twitch_types 0.3.7
- fix non-gated arbitrary
- release twitch_types 0.3.8
- fix...
- release twitch_types 0.3.9
- remove actions-rs
- bump aliri_braid
- release twitch_types 0.3.10
- fix badge
- update checkout
- Add renovate.json
- Update Swatinem/rust-cache action to v2
- release twitch_types 0.3.11
- remove aliri_braid, thiserror, displaydoc and serde_repr, also make serde optional
- add export
- Rename `IntoCow::to_cow` -> `into_cow`
- improve intocow
- changelog
- release twitch_types 0.4.0
- add charity donation id
- release twitch_types 0.4.1
- f bors
- don't check audit on every push
- make all jobs needed
- add prettier
- ensure msrv is correct in ci workflows
- add CCL
- release twitch_types 0.4.2
- Update actionsx/prettier digest to 3d9f7c3
- apply prettier
- make clippy happy
- clippy has a false-positive on nightly
- make it so that xtask isn't needed on msrv check
